services/database/program.ts

```diff
@@ -58,4 +56,18 @@ async function deleteProgram(id: string): Promise<Program> {
     });
     return program;
 }
-export { getProgram, getPrograms, createProgram, deleteProgram };
+
+async function updateProgram(
+    id: string,
+    updatedProgramData: ProgramInput,
+): Promise<Program> {
+    const program = await prisma.program.update({
+        where: {
+            id: parseInt(id),
```

Comment:
this probably doesn't matter in our case but what happens if id isn't a parseable int?

Reply:
Pretty sure it'll just return an error if it isn't an int

Reply:
Worth testing though to verify @victoranyun

Reply:
If a float is passed in, parseInt rounds down and returns the integer, for example `parseInt(2.5)` gives you 2, the error is returned if that integer doesn't correspond to an existing record. 

Should we fix this in a separate PR that throws an error when floats are passed in?

Reply:
We could also do this validation in our validation function. @brandonwong368  We can fix it in a different PR.

Reply:
Okay sounds good